.travis.yml

```diff
@@ -22,6 +22,7 @@ rvm:
   - 2.0
   - 2.1
   - 2.2
+  - 2.3.1
```

Comment:
Is it possible to instead just use `2.3` so it automatically tests against the latest version, or does RVM/Travis then complain it can't find the right version?